sts-persistence-jpa/src/main/resources/db/migration/liquibase/changelog.yml

```diff
@@ -15,3 +15,9 @@ databaseChangeLog:
     file: db/migration/liquibase/changelogs/06-create-table-secret.yml
 - include:
     file: db/migration/liquibase/changelogs/07-enlarge-secret-column_in-table-secret.yml
+- include:
+    file: db/migration/liquibase/changelogs/08-create_table_sts_lock.yml
+- include:
+    file: db/migration/liquibase/changelogs/09-migrate-lock_persistence.yml
+- include:
+    file: db/migration/liquibase/changelogs/10-drop_table_lock_persistance.yml
```

Comment:
if migration will be failed or something would be wrong, does this script would run